source/components/organisms/PinInputModal/PinInputModal.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import Button from "../../atoms/Button";
+import { Heading, Input, Text } from "../../atoms";
+import Wrapper from "../../molecules/Dialog/Wrapper";
+import { Modal } from "../../molecules/Modal";
+import { BackgroundBlur } from "../../atoms/BackgroundBlur";
+import { ButtonContainer, DialogContainer } from "./styled";
+
+export interface PinInputModalProps {
+  visible: boolean;
+  mainApplicantName: string;
+  onClose(): void;
+  onPinEntered(pin: string): void;
+  error?: string;
+}
+
+export default function PinInputModal({
+  visible,
+  mainApplicantName,
```

Comment:
How about changing the property name from `mainApplicantName`to only `name`? By doing that we do not say that this modal is only for displaying main applicant name but instead any name we provide.

Reply:
Sure, only thing is that it might not be clear what `name` is used for, but it really isn't now either so I'll change it.